### DISCUSSION TITLE
Will using PCM affect machine performance?

### DISCUSSION THREAD

**Discussion by jasonneverstop** [opcm/pcm#554] [General]
Will using PCM affect machine performance? What is the recommended sampling interval and parameter settings?

**Comment by ogbrugge** [comment]
Hi,

yes there is always a performance impact but when you use roughly 1 sample second intervals and with that it should not really impact performance. Less than that will introduce overhead that is noticeable in the performance. We normally use a sample frequency of 1 second, this leaves the sample overhead to less than 5%. Using a sample interval of less than 0.1 second is not recommended.

Hope this helps.
